staging/src/k8s.io/apimachinery/pkg/runtime/serializer/json/json.go

```diff
@@ -144,10 +145,10 @@ func (customNumberDecoder) Decode(ptr unsafe.Pointer, iter *jsoniter.Iterator) {
 	}
 }
 
-// CaseSensitiveJsonIterator returns a jsoniterator API that's configured to be
```

Comment:
Technically these are breaking changes, but since you didn't have to update client-go, and we don't offer a promise on this, it's OK. Just FYI that if we ever start being good citizens for this library, we won't be able to make changes like this.